packages/frontend-shared/cypress/e2e/support/e2eSupport.ts

```diff
@@ -421,6 +421,8 @@ function validateExternalLink (subject, options: ValidateExternalLinkOptions | s
   })
 }
 
+Cypress.on('uncaught:exception', (err) => !err.message.includes('ResizeObserver loop limit exceeded'))
```

Comment:
This is here because in CI there were 16 tests failing due to the error. The error itself is safe to ignore in that it doesn't represent something that blows up the UI and it's debatable that it should be an error instead of a warning. I'm open to suggestions for handling this differently though.

Reply:
This exception has been a pain in the butt [forever](https://stackoverflow.com/questions/49384120/resizeobserver-loop-limit-exceeded).